test/spec-directory/iteration.test.ts

```diff
@@ -0,0 +1,60 @@
+import path from "path"
+import { fromPath, SpecDirectory } from "../../lib-js/spec-directory"
+
+describe("SpecDirectory iteration", () => {
```

Comment:
It looks like these tests only test one of real directory/virtual directory, even though the underlying implementation differs between them. It's probably a good idea to test both.